docs/current/README.md

```diff
@@ -18,98 +18,130 @@
+
+The only thing you need pre-installed to run Vaquero is [Docker](https://www.docker.com/).
+
+See the [Getting Started](https://ciscocloud.github.io/vaquero-docs/docs/current/getting-started.html) page for details on deploying Vaquero in virtualbox.
 
 # [Architecture](https://ciscocloud.github.io/vaquero-docs/docs/current/architecture.html)
-![](https://raw.githubusercontent.com/CiscoCloud/vaquero-docs/gh-pages/docs/current/ppt-arch.png)
+![](https://raw.githubusercontent.com/CiscoCloud/vaquero-docs/gh-pages/docs/current/arch-revision.png)
 
-## [Data Model Templates In Depth](https://ciscocloud.github.io/vaquero-docs/docs/current/data-model-howto.html)
-Data Models are used by vaquero as the "Source of Truth" to describe your data center. Data Models define machine operating systems, subnets and boot scripts. We provide some [example data models](https://github.com/CiscoCloud/vaquero-examples) as a reference to build your own.
+## [Data Model Templates](https://ciscocloud.github.io/vaquero-docs/docs/current/data-model-howto.html)
+Data Models are used by Vaquero as the "Source of Truth" to describe your data center. Data Models define machine operating systems, subnets, and boot scripts. We provide some [example data models](https://github.com/CiscoCloud/vaquero-examples) as a reference to build your own.
 
-Notable branches in the example repo:
+Two notable branches in the example repo:
 
 - [`master`](https://github.com/CiscoCloud/vaquero-examples): This will be updated to reflect a complete data model for reference. We will keep this single branch updated when an example of every supported feature, model type, and workflow is up.
 
-- [`local`](https://github.com/CiscoCloud/vaquero-examples/tree/local): Used for CI functional testing. We currently test at two commits in the history. [Init](https://github.com/CiscoCloud/vaquero-examples/commit/3d0df2db8f04eaeaa30e0542d42aa9d861324e4e) and [Update](https://github.com/CiscoCloud/vaquero-examples/commit/b228c2291c3ae87685b25d1435bfe450bf40456b).
-
 - [`vagrant`](https://github.com/CiscoCloud/vaquero-examples/tree/vagrant): Used for small deployments via vagrant in virtualbox. This branch may not show every feature, but it will be leveraged as a small scale example Data Model to deploy a few machines at most.
 
-## [Project Requirements](https://ciscocloud.github.io/vaquero-docs/docs/current/requirements.html)
-
 ## Configuring and Running Vaquero
-Vaquero can run in multiple modes: `server`, `agent`, and `standalone`. This configuration is for standalone mode, which runs server and agent in the same container. See the [architecture page](https://ciscocloud.github.io/vaquero-docs/docs/current/architecture.html) for more details about server and agent.
+![](https://raw.githubusercontent.com/CiscoCloud/vaquero-docs/gh-pages/docs/current/many-vaqueros.png)
```

Comment:
lets put a description that this is what a production deployment would look like